src/coreclr/pgosupport.cmake

```diff
@@ -1,5 +1,18 @@
-include(CheckIPOSupported)
-check_ipo_supported(RESULT HAVE_LTO)
+include(CheckCXXSourceCompiles)	
```

Comment:
nit: all lines in this file have an extra `tab` character at the end of the line